### PR TITLE
duplicity: upgrade to pyrax 1.9.7

### LINKED ISSUE #49498: Please update pyrax in duplicity bottle
`duplicity` bottle comes with `pyrax` 1.9.6 (at this time 1.9.7 is available).

The problem is that `pyrax` 1.9.6 refers to `novaclient.v1_1` which is not included in `python_novaclient` 3.2.0 bundled in `duplicity` bottle.

Is there a simple way to generate `duplicity` bottle with `pyrax` 1.9.7

PS: a workaround is to install pyrax using pip and patching the `/usr/local/Cellar/duplicity/0.7.06_1/bin/duplicity` wrapper:

```
pip install pyrax
```

```diff
--- /usr/local/Cellar/duplicity/0.7.06_1/bin/duplicity.orig	2016-02-24 23:21:03.000000000 +0100
+++ /usr/local/Cellar/duplicity/0.7.06_1/bin/duplicity	2016-02-24 23:21:33.000000000 +0100
@@ -1,2 +1,2 @@
 #!/bin/bash
-PYTHONPATH="/usr/local/Cellar/duplicity/0.7.06_1/libexec/vendor/lib/python2.7/site-packages:/usr/local/Cellar/duplicity/0.7.06_1/libexec/lib/python2.7/site-packages" exec "/usr/local/Cellar/duplicity/0.7.06_1/libexec/bin/duplicity" "$@"
+PYTHONPATH="/usr/local/lib/python2.7/site-packages:/usr/local/Cellar/duplicity/0.7.06_1/libexec/vendor/lib/python2.7/site-packages:/usr/local/Cellar/duplicity/0.7.06_1/libexec/lib/python2.7/site-packages" exec "/usr/local/Cellar/duplicity/0.7.06_1/libexec/bin/duplicity" "$@"
```

### PR DESCRIPTION
**Pull request by renard**
Ref: #49498

Signed-off-by: Sébastien Gross <seb•ɑƬ•chezwam•ɖɵʈ•org>

### CODE REVIEW

File: Library/Formula/duplicity.rb [hunk 1]
```diff
@@ -319,6 +319,8 @@ def install
       r.stage { system "python", *Language::Python.setup_install_args(libexec/"vendor") }
     end
 
+    chmod 0644, Dir[vendor_site_packages/"prettytable-*-py*.egg-info/*"]
```
Comment (DomT4) [inline Library/Formula/duplicity.rb]: Is this a problem for the `pip install`'d version as well? If so, has it been reported upstream?

Comment (renard) [inline Library/Formula/duplicity.rb]: The pip does not really installs the same files:

PIP:
````
-rw-r--r--  1 _brew  _brew     10 Feb 26 15:14 /usr/local/lib/python2.7/site-packages/prettytable-0.7.2.dist-info/DESCRIPTION.rst
-rw-r--r--  1 _brew  _brew    700 Feb 26 15:14 /usr/local/lib/python2.7/site-packages/prettytable-0.7.2.dist-info/METADATA
-rw-r--r--  1 _brew  _brew    606 Feb 26 15:14 /usr/local/lib/python2.7/site-packages/prettytable-0.7.2.dist-info/RECORD
-rw-r--r--  1 _brew  _brew     92 Feb 26 15:14 /usr/local/lib/python2.7/site-packages/prettytable-0.7.2.dist-info/WHEEL
-rw-r--r--  1 _brew  _brew    821 Feb 26 15:14 /usr/local/lib/python2.7/site-packages/prettytable-0.7.2.dist-info/metadata.json
-rw-r--r--  1 _brew  _brew     12 Feb 26 15:14 /usr/local/lib/python2.7/site-packages/prettytable-0.7.2.dist-info/top_level.txt
-rw-r--r--  1 _brew  _brew  54204 Feb 26 15:14 /usr/local/lib/python2.7/site-packages/prettytable.py
-rw-r--r--  1 _brew  _brew  52667 Feb 26 15:14 /usr/local/lib/python2.7/site-packages/prettytable.pyc
````

The duplicity install:

````
-rw-------  1 _brew  _brew    710 Dec  7 13:03 /usr/local/Cellar/duplicity/0.7.06_2/libexec/vendor/lib/python2.7/site-packages/prettytable-0.7.2-py2.7.egg-info/PKG-INFO
-rw-------  1 _brew  _brew    230 Dec  7 13:03 /usr/local/Cellar/duplicity/0.7.06_2/libexec/vendor/lib/python2.7/site-packages/prettytable-0.7.2-py2.7.egg-info/SOURCES.txt
-rw-------  1 _brew  _brew      1 Dec  7 13:03 /usr/local/Cellar/duplicity/0.7.06_2/libexec/vendor/lib/python2.7/site-packages/prettytable-0.7.2-py2.7.egg-info/dependency_links.txt
-rw-------  1 _brew  _brew     12 Dec  7 13:03 /usr/local/Cellar/duplicity/0.7.06_2/libexec/vendor/lib/python2.7/site-packages/prettytable-0.7.2-py2.7.egg-info/top_level.txt
-rw-r--r--  1 _brew  _brew  54204 Dec  7 13:03 /usr/local/Cellar/duplicity/0.7.06_2/libexec/vendor/lib/python2.7/site-packages/prettytable.py
````

As you can see the install is not quite the same. In the duplicity install at leat the PKG-INFO shoulf be readable.

**Comment by tdsmith** [comment]
momentarily blocking on #49710

**Comment by apjanke** [comment]
#49710 has been merged. Want to close this out and re-submit on the new homebrew/core?

**Comment by apjanke** [comment]
It looks like duplicity is already bumped to use Pyrax 1.9.7, via https://github.com/Homebrew/homebrew-core/commit/89ff11d3dd68bf470a43921574ad6c4137225fcd. Closing this out.

**Pull request closed without merging**